### DISCUSSION TITLE
Report Job Timeout

### DISCUSSION THREAD

**Discussion by willflo** [netbox-community/netbox#11124] [Q&A]
Hey Everybody,

i cant seem to figure this issue out. I do have a report job running manually which times out after 300 seconds. This seems to be a default timeout.
Can anyone tell me how i put this timeout higher so the job doesnt run into the emergency stop?
This error is: 
An exception occurred: JobTimeoutException: Task exceeded maximum timeout value (300 seconds)

Traceback (most recent call last):
rq.timeouts.JobTimeoutException: Task exceeded maximum timeout value (300 seconds)

Thank you.

**Comment by kkthxbye-code** [comment]
It's in the docs:

https://docs.netbox.dev/en/stable/configuration/miscellaneous/#rq_default_timeout

or 

https://docs.netbox.dev/en/stable/customization/reports/#job_timeout

**Reply by bitcollector1** [reply]
thank you very much for this, I've been banging my head all day on this :)  I guess I really should write my scripts to more efficient ;)
